Resources/Prototypes/Entities/Structures/Dispensers/base_structuredispensers.yml

```diff
@@ -62,7 +62,12 @@
     - 0,0,19,5
   - type: ReagentDispenser
     beakerSlot:
-      whitelistFailPopup: reagent-dispenser-component-cannot-put-entity-message
+      # TODO: There's a problem here in that item slots are checked before
+      # storage containers when inserting... so if you insert a beverage jug
+      # you see the item slot popup even though it inserts into the storage!
+      # It'd be nice to have the popup only show up if it can't fit in storage.
+      # Also the storage comp should have not the default insert sound.
+      # whitelistFailPopup: reagent-dispenser-component-cannot-put-entity-message
```

Comment:
preexisting bug, doesn't seem to be worsened by this PR?

Reply:
Fair point; it was one of those things where I assumed I had caused it and only realized later on that it was preexisting.

I'll leave the comment but uncomment the popup, I guess.